src/main/java/com/paneedah/weaponlib/EntityProjectile.java

```diff
@@ -234,7 +237,7 @@ public void onUpdate() {
          *  GLASS BREAK CHECK
          */
 
-        if(ModernConfigManager.bulletBreakGlass && !world.isRemote) {
+        if((ModernConfigManager.bulletBreakGlass && !world.isRemote) || ModernConfigManager.penetratesGlass) {
```

Comment:
This would cause a crash do `(!world.isRemote && (ModernConfigManager.bulletBreakGlass || ModernConfigManager.penetratesGlass))` instead